NDTensors/ext/NDTensorsMetalExt/imports.jl

```diff
@@ -1,5 +1,6 @@
 import NDTensors: mtl, set_ndims, set_eltype, set_eltype_if_unspecified
 import NDTensors.SetParameters: nparameters, get_parameter, set_parameter, default_parameter
+import NDTensors.Unwrap: Exposed
```

Comment:
You should use `using` here instead of `import`, `import` should only be used if you are overloading a function, as opposed to just making it available in the namespace.